src/mds/Locker.h

```diff
@@ -36,7 +36,7 @@ class CDentry;
 class Capability;
 class SimpleLock;
 class ScatterLock;
-class LocalLock;
+class LocalLockCC;
```

Comment:
Typo here?
It's also probably redundant since it doesn't seem to be breaking the build, but better safe than not.

Reply:
Indeed, thanks for spotting it.